addons/udes_warehouse_classification/wizard/add_product_classification.py

```diff
@@ -11,6 +11,7 @@ class AddProductClassification(models.TransientModel):
     """Add product classification action wizard."""
 
     _name = "udes_warehouse_classification.add_product_classification"
+    _description = "Warehouse Classification - Add product classification wizard"
```

Comment:
Nit: Maybe name should be shorter as name of model can be display name of many2one field of model.model i think.

Reply:
It won't be shown as m2o, it's a wizard. Ignoring :smile: